telnyx_rtc/src/main/java/com/telnyx/webrtc/sdk/TelnyxClient.kt

```diff
@@ -966,14 +979,15 @@ class TelnyxClient(
 
     /**
      * Sets the global SDK log level
-     * Logging is implemented with Timber
+     * Logging is implemented with Logger
```

Comment:
'Logging is implemented with the Logger provided via the [Config], if none is provided then the default logger in [TxLogger] is used'

Reply:
Ideally the stuff in the square brackets links to the actual class 😅  @isaacakakpo1